app/assets/javascripts/jquery.dualpane.min.js

```diff
@@ -0,0 +1 @@
+!function(a){a.fn.dualpane=function(b){function j(){f.css("width",d.outerWidth()-e.outerWidth()-g.outerWidth()+"px")}function k(a){return 0!==a%1?a=100*a+"%":(a>=d.outerWidth()-c.limit&&(a=d.outerWidth()-c.limit),a+="px"),a}var c=a.extend({},a.fn.dualpane.defaults,b);this.addClass("jquery-dualpane");var d=this,e=this.children().first(),f=this.children().last(),g=a('<div class="jquery-dualpane-dragbar"></div>');if(e.after(g),c.rightColWidth){var h=k(c.rightColWidth);f.css("width",h),e.css("width",d.outerWidth()-f.outerWidth()-g.outerWidth()+"px")}else{var i=k(c.leftColWidth);e.css("width",i),j()}return g.on("mousedown touchstart",function(b){b.preventDefault(),a(document).bind("mousemove touchmove",function(a){var b;b="touchmove"==a.type?a.originalEvent.touches[0].clientX:a.clientX;var f=b-e.offset().left;f>=c.limit&&f<=d.outerWidth()-c.limit&&(e.css("width",b-e.offset().left+"px"),j())}).bind("mouseup touchend",function(){a(document).unbind("mousemove touchmove")})}),a(window).on("resize",function(){var a=e.outerWidth()/d.outerWidth(),b=a*d.outerWidth();b=b<c.limit?c.limit+"px":b>d.outerWidth()-c.limit?d.outerWidth()-c.limit+"px":100*a+"%",e.css("width",b),j()}),this},a.fn.dualpane.defaults={leftColWidth:.5,limit:100}}(jQuery);
```

Comment:
Line is too long.<br>Missing semicolon.<br>Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.<br>A leading decimal point can be confused with a dot: '.5'.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.